review/communication-react.api.md

```diff
@@ -0,0 +1,2130 @@
+    disposeView(callId: string | undefined, participantId: CommunicationIdentifier | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState): void;
+    getState(): CallClientState;
+    offStateChange(handler: (state: CallClientState) => void): void;
+    onStateChange(handler: (state: CallClientState) => void): void;
+}
+
+// @public
+export type StatefulCallClientArgs = {
+    userId: CommunicationUserIdentifier;
+};
+
+// @public
+export type StatefulCallClientOptions = {
+    maxStateChangeListeners?: number;
+};
+
+// @public
+export interface StatefulChatClient extends ChatClient {
+    getState(): ChatClientState;
+    offStateChange(handler: (state: ChatClientState) => void): void;
+    onStateChange(handler: (state: ChatClientState) => void): void;
+}
+
+// @public
+export type StatefulChatClientArgs = {
+    userId: CommunicationUserIdentifier;
+    displayName: string;
+    endpoint: string;
+    credential: CommunicationTokenCredential;
+};
+
+// @public
+export type StatefulChatClientOptions = {
+    chatClientOptions: ChatClientOptions;
+    maxStateChangeListeners?: number;
+};
+
+// @public
+export interface StatefulDeviceManager extends DeviceManager {
+    selectCamera: (VideoDeviceInfo: any) => void;
+}
+
+// @public
+export const StreamMedia: (props: StreamMediaProps) => JSX.Element;
+
+// @public
+export interface StreamMediaProps {
+    isMirrored?: boolean;
+    styles?: BaseCustomStyles;
+    videoStreamElement: HTMLElement | null;
+}
+
+// @public
+export type SystemMessage = ParticipantAddedSystemMessage | ParticipantRemovedSystemMessage | TopicUpdatedSystemMessage | ContentSystemMessage;
+
+// @public
+export interface SystemMessageCommon extends MessageCommon {
+    // (undocumented)
+    iconName: string;
+    // (undocumented)
+    messageType: 'system';
+}
+
+// @public
+export const toFlatCommunicationIdentifier: (identifier: CommunicationIdentifier) => string;
+
+// @public
+export type TopicChangedListener = (event: {
+    topic: string;
+}) => void;
+
+// @public
+export interface TopicUpdatedSystemMessage extends SystemMessageCommon {
+    // (undocumented)
+    systemMessageType: 'topicUpdated';
+    // (undocumented)
+    topic: string;
+}
+
+// @public
+export interface TranscriptionCallFeature {
+    isTranscriptionActive: boolean;
+}
+
+// @beta
+export interface Transfer {
+    error?: TransferErrorCode;
+    id: number;
+    state: TransferState;
+    targetParticipant: CommunicationUserIdentifier | PhoneNumberIdentifier | MicrosoftTeamsUserIdentifier | UnknownIdentifier;
+}
+
+// @beta
+export interface TransferCallFeatureState {
+    receivedTransferRequests: TransferRequest[];
+    requestedTransfers: Transfer[];
+}
+
+// @beta
+export interface TransferRequest {
+    // (undocumented)
+    targetParticipant: CommunicationUserKind | PhoneNumberKind | MicrosoftTeamsUserKind | UnknownIdentifier;
```

Comment:
Is this fine that we are mixing UnknownIdentifier with these Kind types?

Reply:
Nope, nice catch. All of them should be the non-kind types.
```typescript
targetParticipant: CommunicationUserIdentifier | PhoneNumberIdentifier | MicrosoftTeamsUserIdentifier | UnknownIdentifier;
```
 => @prprabhu-ms 
If this is a bug in Calling, let's tell them to change it asap while it's still in beta.

Reply:
It's a bug in calling:
![image](https://user-images.githubusercontent.com/1885005/142190081-a149d5e7-691e-4fdc-a07e-cff359cb5ab9.png)